cmd/main_test.go

```diff
@@ -101,10 +101,13 @@ internalCertManagement:
 				LeaderElection:             false,
 				LeaderElectionResourceLock: "changed",
 				LeaderElectionID:           "changed",
+				LeaderElectionNamespace:    "test-ns",
 				LeaseDuration:              ptr.To(1 * time.Minute),
 				RenewDeadline:              ptr.To(1 * time.Minute),
 				RetryPeriod:                ptr.To(1 * time.Minute),
 				HealthProbeBindAddress:     ":9443",
+				LivenessEndpointName:       "/healthz",
```

Comment:
This is independent from the changes in this PR. Actually, without this assignment, unit tests should be failing on main branch (it fails on my local).  I'm not sure there is an issue in testing side?

Reply:
Because `cmd` is not included in the test, do you mean adding it please? https://github.com/kubernetes-sigs/lws/blob/main/Makefile#L113

Reply:
@ahg-g  Thank you for your guidance. Unit tests under cmd directory will be executed in `make test` from now on.